source/megengine.js

```diff
@@ -415,6 +411,10 @@ megengine.Graph = class {
                 const argument = new megengine.Argument(name, type, initializer, quantization);
                 args.push(argument);
             }
+            else if(opr.shape) {
+                const type = new megengine.TensorType('TypePlaceHolder', new megengine.TensorShape(opr.shape));
```

Comment:
What is `TypePlaceHolder`?

Reply:
This is a modification to the first bug. Sometimes our node has no tensor information but only shape information (that is, if-else here). At this time, there will be no type, but we need to display the shape, so typeplaceholder is used here to indicate this situation.

Reply:
Would it work to use `'?'` as some of the other frameworks?

Reply:
yes, that's a good idea. done.